docs/components/overview.md

```
@@ -14,58 +14,53 @@ import { CategoryButton } from '@site/src/components/CategoryButton';
 
 <IxLayoutGrid style={{marginBottom: '3rem'}} no-margin>
-<IxRow style={{gap: '0.5rem', justifyContent: 'center'}}>
-<CategoryButton
+  <IxRow style={{gap: '0.5rem', justifyContent: 'center'}}>
+    <CategoryButton
     title="Application frame"
     link="#application-frame">
-![Application frame](https://www.figma.com/design/wEptRgAezDU1z80Cn3eZ0o/iX-Documentation-illustrations?node-id=5186-249&t=gkh6VNlJun96I6Ac-4)
-</CategoryButton>
-<CategoryButton
+![](https://www.figma.com/design/wEptRgAezDU1z80Cn3eZ0o/iX-Documentation-illustrations?node-id=5186-249&t=gkh6VNlJun96I6Ac-4)
```

markdownlint-cli2 (0.23.2)

[warning] Images should have alternate text (alt text)

(MD045, no-alt-text)